applications/nrf5340_audio/src/bluetooth/le_audio.h

```diff
@@ -127,13 +127,24 @@ struct le_audio_evt {
 typedef void (*le_audio_receive_cb)(const uint8_t *const data, size_t size, bool bad_frame,
 				    uint32_t sdu_ref);
 
+enum le_audio_button_id {
+	LE_AUDIO_BUTTON_ID_1,
```

Comment:
The buttons have little to do with AUDIO, so I suggest to remove "LE_AUDIO". 
This seems like a boardfile define?

Reply:
You have a point, but if we do this the enum will need to move to the audio module. Perhaps it would be better to replace "BUTTON" with "USER_ACTION"?

Reply:
I was also thinking of using boardfile define, but we do not want to have the le_audio_xxx files being dependent on a board file.
Changed to LE_AUDIO_USER_DEFINED_ACTION